qemu/tests/balloon_check.py

```diff
@@ -424,6 +425,8 @@ def run(test, params, env):
             balloon_type = params_tag['balloon_type']
             min_sz, max_sz = balloon_test.get_memory_boundary(balloon_type)
             expect_mem = int(random.uniform(min_sz, max_sz))
+            if params_tag.get('small_value_check', 'no') == 'yes':
```

Comment:
`minimum_value_check` would be more accurate.

Reply:
Modified,thanks